src/effects/effectsmanager.cpp

```diff
@@ -70,8 +70,9 @@ EffectsManager::~EffectsManager() {
 
 bool alphabetizeEffectManifests(EffectManifestPointer pManifest1,
                                 EffectManifestPointer pManifest2) {
-    int dNameComp = QString::localeAwareCompare(pManifest1->displayName(), pManifest2->displayName());
+    // Sort built-in effects first before external plugins
     int bNameComp = static_cast<int>(pManifest1->backendType()) - static_cast<int>(pManifest2->backendType());
+    int dNameComp = QString::localeAwareCompare(pManifest1->displayName(), pManifest2->displayName());
```

Comment:
What do `b` and `d` before `NameComp` mean here? Backend and... ? Let's use variable names that explain themselves better. how about "backendNameComparison" and "displayNameComparison"?